src/main/resources/graphql/service/query.graphqls

```diff
@@ -59,3 +64,23 @@ type ContentPayload {
     pageInfo: PaginationInfo!
 }
 
+type CompositeProgressInformation {
+
+    """
+    ID of chapter
+    """
+    chapterId: UUID!
```

Comment:
I said I want to use this type for the course progress as well, so please remove the chapter Id from it

Reply:
I don't think it's needed by the frontend anyways, if they fetch the chapter progress for a chapter they obviously know what chapter it is for